### PR TITLE
Updated animalfacts.py with animal facts

### PR DESCRIPTION
**Pull request by tanushree57**
Updated LOBSTER FACTS added 2 lines

### CODE REVIEW

File: animalfacts.py [hunk 1]
```diff
@@ -1771,6 +1771,8 @@ def animalfactsbot(reddit):
     'Lobsters have poor eyesight, but have highly developed senses of smell and taste.',
     'Lobsters taste with their legs and chew with their stomachs.',
     'Lobsters keep growing forever. They do not get weaker or lose their ability to reproduce, and will keep on molting and growing.',
+    'Lobsters are invertebrates which means they lack a backbone.'
+    'Lobsters dont have brains as their nervous system is very primitive.'
```
Comment (mpentler) [inline animalfacts.py]: Actually, would you mind putting a comma in here so it matches the other tuples?

**Comment by mpentler** [comment]
Oh, apparently you'll need to pull the latest too, I've been merging PRs for the past hour or so. Sorry!

**Comment by tanushree57** [comment]
> Oh, apparently you'll need to pull the latest too, I've been merging PRs for the past hour or so. Sorry!

Haha alright. I'm on it now.

File: animalfacts.py (hunk 1 above)
Comment (tanushree57) [reply animalfacts.py]: Done! :D

Comment (mpentler) [reply animalfacts.py]: Thanks! That means I can fix this new conflict more easily. There were two other lobster facts added and now yours slot right in.

**Comment by tanushree57** [comment]
@mpentler no problem! :D

**Pull request merged**